src/pkg/selectors/exchange_test.go

```diff
@@ -805,7 +809,7 @@ func (suite *ExchangeSelectorSuite) TestIdPath() {
 		},
 	}
 	for _, test := range table {
-		suite.T().Run(test.cat.String(), func(t *testing.T) {})
+		suite.Run(test.cat.String(), func() {})
```

Comment:
@ryanfkeepers This test isn't actually running anything. Can we file an issue/make a PR to fix that?

Reply:
mmm, lets just delete the test.  I'm not even sure what it's supposed to be testing, and I have a suspicion that it's vestigial.

Reply:
see #2665